include/deal.II/distributed/tria_base.h

```diff
@@ -304,6 +304,15 @@ namespace parallel
     virtual types::coarse_cell_id
     n_global_coarse_cells() const override;
 
+    /**
+     * Reset this triangulation into a virgin state by deleting all data.
```

Comment:
Let's avoid the void "virgin". Can you change it to "to an empty state"? I will update all of the other places in the library that use the word.

Reply:
Ahh you know what, that's such a good idea. Reminds me of the main/master thing.
Small changes that make no real difference, but makes everyone safer and happier.

Reply:
Done :+1: